### PR TITLE
Add second vaccination astraZeneca

### PR DESCRIPTION
**Pull request by frauhottelmann**
I hope I didn't forget anything 😄

### CODE REVIEW

File: src/data-requests/vaccination.ts [hunk 1]
```diff
@@ -96,9 +98,10 @@ export async function getVaccinationCoverage(): Promise<ResponseData<Vaccination
         secondVaccinated: number,
         secondbiontech: number,
         secondmoderna: number,
+        secondAstraZeneca: number,
         secondDifference: number,
         secondquote: number
-    }>(sheet, { header: ["ags", "state", "administeredVaccinations", "firstVaccinated", "firstbiontech", "firstmoderna", "firstAstraZeneca", "firstDifference", "firstquote", "secondVaccinated", "secondbiontech", "secondmoderna", "secondDifference", "secondquote"], range: "A4:N21" })
+    }>(sheet, { header: ["ags", "state", "administeredVaccinations", "firstVaccinated", "firstbiontech", "firstmoderna", "firstAstraZeneca", "firstDifference", "firstquote", "secondVaccinated", "secondbiontech", "secondmoderna", "secondAstraZeneca", "secondDifference", "secondquote"], range: "A4:N21" })
```
Comment (Rubber1Duck) [inline src/data-requests/vaccination.ts:105]: range: "A4:N21" must be incresed to range: "A4:O21"!!

**Comment by Rubber1Duck** [comment]
And dont forget the docs!

**Comment by Rubber1Duck** [comment]
looks fine for me!

**Comment by Rubber1Duck** [comment]
hopefully marlon will merge this request soon ...

**Comment by Rubber1Duck** [comment]
tested on my local server, it works fine

**Review by marlon360 — APPROVED** [review]

**Comment by marlon360** [comment]
Thanks @frauhottelmann for this PR, looks good!
And thank you @Rubber1Duck for reviewing. Great job!

**Pull request merged**